README_JP.md

```diff
@@ -0,0 +1,290 @@
+例えば、ATS プラグインを `Scenarios/someone/sometrain/ats32/plugin.dll`、DXDynamicTexture (`Zbx1425.DXDynamicTexture-net35.dll`) を `Scenarios/zbx1425/Zbx1425.DXDynamicTexture-net35.dll` に配置する場合は、 `../../../zbx1425` を指定します。
+
+次に、ATS プラグインの Load() メソッドに TextureManager を初期化するコードを追加します。
+
+```csharp
+// using Zbx1425.DXDynamicTexture;
+
+[DllExport(CallingConvention.StdCall)]
+public static void Load() {
+    TextureManager.Initialize();
+}
+```
+
+ここまで完了したら、ATS プラグインをコンパイルして BVE から読み込み、エラーが発生しないことを確認してください。
+
+何か問題があれば、お気軽に E メールか Twitter からお問い合わせください。
+
+## 動的テクスチャ
+
+コード例 (重要な部分のみであり、完全なコードではありません) :
+
+```csharp
+// using System;
+// using System.Drawing;
+// using System.IO;
+// using System.Reflection;
+// using Zbx1425.DXDynamicTexture;
+
+private static TextureHandle hTex;
+private static GDIHelper gClock;
+private static Bitmap imgClock;
+private static string dllParentPath = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
+
+[DllExport(CallingConvention.StdCall)]
+public static void Load() {
+    TextureManager.Initialize();
+    hTex = TextureManager.Register("clock_back_tex.png", 128, 128);
+    gClock = new GDIHelper(128, 128);
+    imgClock = new Bitmap(Path.Combine(dllParentPath, "clock_back.png"));
+}
+
+[DllExport(CallingConvention.StdCall)]
+public static void Dispose() {
+    TextureManager.Dispose();
+    imgClock.Dispose();
+    gClock.Dispose();
+}
+
+[DllExport(CallingConvention.StdCall)]
+public static AtsHandles Elapse(AtsVehicleState state, IntPtr hPanel, IntPtr hSound) {
+    var panel = new AtsIoArray(hPanel); var sound = new AtsIoArray(hSound);
+    if (hTex.IsCreated) {
+        panel[100] = 0;
+        if (hTex.ShouldUpdate(state.Time, 5)) {
+            gClock.BeginGDI();
+            gClock.DrawImage(imgClock, 0, 0);
+            gClock.EndGDI();
+
+            gClock.Graphics.DrawString(
+                (state.Time / 1000).ToString(), 
+                new Font(SystemFonts.DefaultFont.FontFamily, 20, GraphicsUnit.Pixel),
+                Brushes.Black, 0, 0
+            );
+
+            hTex.Update(gClock);
+        }
+    } else {
+     	panel[100] = 1;
+    }
+}
+```
+
+この例では、"clock_back_tex.png" という名前の画像ファイルから作成されたテクスチャへ、DLL と同じディレクトリに存在する "clock_back.png" と現在の日時を表す文字列を描画しています。失敗した場合は運転台パネルで使用可能な値 ats100 を 1 に設定しています。
+
+ここからこのコードの解説をしていきます。
+
+- TextureManager.Register(texturePathSuffix*, width*, *height*);
+
+  シナリオの読込中、`texturePathSuffix` で指定したファイル名の画像からテクスチャが生成されるかどうか監視します。
+  戻り値は TextureHandle 型で、テクスチャが生成された場合は TextureHandle.IsCreated プロパティ (コード例の場合は `hTex.IsCreated`) が true になります。 
+  
+  このメソッドを Load メソッド内で呼び出し、Started メソッド内か Elapse メソッド内で TextureHandle.IsCreated プロパティを確認してください。
+  
+  また、このメソッドは空白のテクスチャで元のテクスチャを置き換えるため注意してください。
+  
+  BVE のマップを車両より先に読み込む仕様の関係で、運転台パネル内のテクスチャは直接置き換えられる一方、マップ内のテクスチャを置き換えるには一度マップを読み込んだ後再度読み込み直す必要があります。
+  そのため、ゲーム開始時に IsCreated が true になっているか確認し、false であれば、コード例のように利用者に対しシナリオを読み込み直す必要がある旨を表示することを推奨します。
```

Comment:
I didn't say it clearly enough in the English version. It is only necessary to check `IsCreated` if you are modifying a texture used in the Map, and there should be no need to check when modifying a texture used in Panel.

Reply:
Changed in [583e0f9](https://github.com/zbx1425/DXDynamicTexture/pull/1/commits/583e0f912956a23a0c7ddc68a0f58f2757de02b1):
```diff
  そのため、
+ マップ内のテクスチャを置き換えている場合は、
  ゲーム開始時に IsCreated が true になっているか確認し、false であれば、コード例のように利用者に対しシナリオを読み込み直す必要がある旨を表示することを推奨します。
```
Does this fix the problem correctly?